test/functional/JIT_Test/src/jit/test/jitt/codecache/TestManager.java

```diff
@@ -69,6 +69,10 @@
 	/*Total number of code caches being used at startup, this is equivalent to -Xjit:numOfCodeCachesOnStartup*/
 	private static int numCodeCacheOnStartup = 0;
 
+	/* list of all input arguments from RuntimemxBean */
+	private static RuntimeMXBean RuntimemxBean = (RuntimeMXBean)ManagementFactory.getRuntimeMXBean();
+	private static List<String> aList=RuntimemxBean.getAllInputArguments();
```

Comment:
can you please use a more descriptive name for this variable

Reply:
It was previously called `alist` in the rest of the code. I feel like keeping the variable name the same would be better in terms of clarity.